x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -503,6 +503,33 @@ metricbeat.modules:
   #    fields: # added to the the response in root. overwrites existing fields
   #      key: "value"
 
+#-------------------------------- IBM MQ Module --------------------------------
+- module: ibmmq
+  metricsets: ['qmgr']
+  period: 10s
+  hosts: ['localhost:9157']
+
+  # This module uses the Prometheus collector metricset, all
+  # the options for this metricset are also available here.
+  metrics_path: /metrics
+
+  processors:
```

Comment:
Maybe a short comment/description here of why we decided to use js processor here could be of worth

Reply:
Added!